app/src/main/java/com/codesoom/assignment/controllers/TaskController.java

```diff
@@ -0,0 +1,53 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.application.NotFoundException;
+import com.codesoom.assignment.domain.Task;
+import com.codesoom.assignment.domain.Tasks;
+import com.sun.net.httpserver.HttpExchange;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/tasks")
+public class TaskController {
+    private Tasks tasks = new Tasks();
+    private Long taskId = 1L;
+
+    @GetMapping
+    public List<Task> getTasks() {
+        return tasks.getTasks();
+    }
+
+    @GetMapping("{id}")
+    public Task getTask(@PathVariable("id") Long id) {
+        return tasks.findTask(id).orElseThrow(NotFoundException::new);
```

Comment:
라인이 길어지지 않도록 한 줄에 `.`이 한 개씩 나타나도록 적절해 개행해 줍시다.
```suggestion
        return tasks.findTask(id)
            .orElseThrow(NotFoundException::new);
```

Reply:
return tasks.findTask(id)
            .orElse(null);
같은 경우에도 개행을 해야할까요?

Reply:
이런 건 소속된 팀 컨벤션 등에 따르는 게 좋겠죠. 여기에서는 컨벤션을 정해놓고 하는 것이 아니니 놔둬도 좋을 것 같습니다. 만약 한 줄에 점 하나씩 하겠다고 독하게 마음먹고 실천하시겠다면 리뷰로 도와 드릴 수 있습니다 :-)